### ISSUE TITLE
Support switch statements on tuples

### ISSUE THREAD

**Issue by initram** [tree-sitter/tree-sitter-c-sharp#99]
currently we do not support this:
```
    switch (a, b) {
      case (1, 2):
        return 1;
      default:
        return 0;
    }
```

I am working on a fix for this, but I thought it would be good to have an issue on it as well.
Found used in roslyn: https://github.com/dotnet/roslyn/blob/cf55f3a58e47298426fa971d3bd9d8857c746c65/src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs#L709

**Comment by initram** [comment]
fixed by #100

**Comment by damieng** [comment]
You probably already knew this but if not: If you put "Fixes #bug-number" in the PR somewhere then merging the PR will auto-close the issue.
